### DISCUSSION TITLE
Problems when looking for Scan Response packets

### DISCUSSION THREAD

**Discussion by AndreaCencio** [ThomasGerstenberg/blatann#173] [General]
I have problems when scanning for Scan Response Packet with blatann and an nrf52840 dongle, it is very hard to see a Scan Response packet in scan report, but with nRF official tool I can clearly send scan response data.

What is the best method to call for scan response packets?

**Comment by ThomasGerstenberg** [comment]
The current implementation will combine the advertising packet with the scan response packet to create a single combined entry for the given peer's address, when using [ScanReportCollection.advertising_peers_found](https://github.com/ThomasGerstenberg/blatann/blob/e0b46fbad9737f3387b0bee6e4f3be734c67d6c4/blatann/gap/advertise_data.py#L340).

If using [ScanReportCollection.all_scan_reports](https://github.com/ThomasGerstenberg/blatann/blob/e0b46fbad9737f3387b0bee6e4f3be734c67d6c4/blatann/gap/advertise_data.py#L352) or iterating the waitable, there's a [packet_type](https://github.com/ThomasGerstenberg/blatann/blob/e0b46fbad9737f3387b0bee6e4f3be734c67d6c4/blatann/gap/advertise_data.py#L292) field where you can check if the packet was a scan response
